client/src/requests.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import type { PlaceProps, ReviewProps } from './types';
+
+const axios = require('axios');
+axios.defaults.baseURL = 'http://localhost:3000';
+
+// We are sending the client's timezone offset to the server to calculate
+// the user's time
+const tzOffset = new Date().getTimezoneOffset() * -60;
+
+export class BackendClient {
+  // Get list of tags matching 'q'
+  getTags = (q: string) => (
+    axios.get('/tags?q=' + q)
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((error) => {
+        console.log(error);
+      })
+  );
+
+  // Get a list of places
+  getPlaces = (q?: string, openNow?: boolean) => {
+    let params: {
```

Comment:
better name for **q** (does it means 'tag'?)

Reply:
`q` is a general search term.
currently it searches **tags** and **places**, but can be expanded, so I decided to name it `q`.